packages/location-field/src/index.tsx

```diff
@@ -735,16 +769,14 @@ const LocationField = ({
       <S.InputGroup>
         {/* location field icon -- also serves as dropdown anchor */}
         <S.Dropdown
+          input={<>{textControl}{clearButton}</>}
```

Comment:
This layout is a bit wonky for me, see image below. Could just be local, but it looks like a `width: 100%` is getting added somewhere to the input, which is clashing with the 30px width on the `DropdownButtonIcon`

<img width="477" alt="image" src="https://user-images.githubusercontent.com/115499534/213577732-a8e45d07-4757-4f9b-ae6c-8dc32f98c375.png">

Reply:
Yep there's some otp-ui changes required. There is a PR there that resolves this! Thanks for spotting